tests/test_annotation_gwascatalog.py

```diff
@@ -56,6 +120,30 @@ def test_annotation_gwascatalog_colons(client):
     assert '_' not in entry["variant"]
     assert '/' not in entry["variant"]
 
+def test_annotation_gwascatalog_metadata(client):
+  params = {
+    "filter": "id in 2,3 and rsid eq 'rs7903146'",
+    "limit": 1,
+    "sort": "pos"
+  }
+  results = client.get("/v1/annotation/gwascatalog/results/",query_string=params)
+
+  assert results.status_code == 200
+
+  result_data = results.json
+  assert "data" in result_data
+  assert len(result_data["data"]) > 0
+
+  found_ids = set()
+  for entry in result_data["meta"]["datasets"]:
+    found_ids.add(entry["id"])
+
+  assert 2 in found_ids
+  assert 3 in found_ids
+
+  for k in "id variant rsid chrom pos ref alt trait trait_group risk_allele risk_frq log_pvalue or_beta genes pmid pubdate first_author study".split():
```

Comment:
Is "autoselect build" something we'd want / be able to test with the sample DB used for testing?

(for example, by looking to see if the metadata contains this information if build is specified as a query param)

Reply:
There are test cases for autoselect build for genes, GWAS cat, and recombination. I can't say they're perfect, but basic tests are at least there. 

https://github.com/statgen/locuszoom-api/blob/7e5f7ad4f9fd21a34fc98cd0bcf6abe3b3527def/tests/test_annotation_gwascatalog.py#L43-L60

Reply:
Ah, I see; thanks! This is a good start- apologies for missing the meaning of that test late on friday night. 👓